lib/active_encode/base.rb

```diff
@@ -13,4 +13,6 @@ class Base
     include EngineAdapter
     include Callbacks
   end
+
+  class NotFound < RuntimeError; end
```

Comment:
Ah, good point.  What do you think about leaving this error class here for the adaptors to use?  Or do you think the errors thrown by the adapters should just be allowed to bubble up?